internal/pkg/aws/ecr/ecr.go

```diff
@@ -167,9 +168,30 @@ func (s Service) DeleteImages(images []Image, repoName string) error {
 	return nil
 }
 
+func (s Service) repoExist(repoName string) (bool, error) {
+	_, err := s.ecr.DescribeRepositories(&ecr.DescribeRepositoriesInput{
+		RepositoryNames: aws.StringSlice([]string{repoName}),
+	})
+	if err != nil {
+		aerr := err.(awserr.Error)
+		if aerr.Code() != "RepositoryNotFoundException" {
```

Comment:
Should we explicitly handle the scenario where `aerr.Code()` **is equal to** `RepositoryNotFoundException` rather than handling everything that is not? I find the current flow is easy to misread if you're not looking closely.

Reply:
addressed